Makefile.in

```diff
@@ -1090,7 +1090,7 @@ $(IBIN_DIR):
 
 # ----------------------------------------------------------------------
 
-.PHONY: clean eclean bootstrap_root_clean bootstrap_clean
+.PHONY: clean eclean bootstrap_root_clean bootstrap_clean test
```

Comment:
Put the .PHONY with the test target, this .PHONY is only for clean targets

Reply:
Fixed in commit below.